scripts/basic-workflow-tests.sh

```diff
@@ -253,48 +257,52 @@ test_taxonomy() {
 
 test_generate() {
     task Generate synthetic data
-    GENERATE_ARGS+=("--endpoint-url" "http://localhost:8000/v1")
+    local generate_args
```

Comment:
what is the GENERATE_ARGS at the top of the file used for?

Reply:
I think it's the same as `TRAIN_ARGS` where it's being used as a base global arg - going to go the same route and leave this be for now